tests/test_utils.py

```diff
@@ -159,22 +161,55 @@
     "bio_things": {"class": "BioThings", "property": "bioThings"},
 }
 
+test_disk_storage = [
+    (2, 4000, 16000),
+    (1000, 4000, 16000),
+    (2000000, 1900000, 2000000),
+    (1073741825, 1073741824, 1181116006.4),
+]
+
+
+# create temporary files with various size based on request
+@pytest.fixture()
+def create_temp_query_file(
+    tmp_path: Path, request: FixtureRequest
+) -> Generator[Tuple[Path, Path, Path], None, None]:
```

Comment:
You cna just use `tuple` here as opposed to `Tuple`

Reply:
Okay I will make that change.